tests/Backend/Workspaces/WorkspacesSnowflakeTest.php

```diff
@@ -836,4 +836,235 @@ public function dataTypesDiffDefinitions()
+        self::assertCount(10, $backend->fetchAll('languages'));
+
+        // load data from workspace to table
+        $workspace2 = $workspaces->createWorkspace([], true);
+        $options = [
+            'input' => [
+                [
+                    'source' => $tableId,
+                    'destination' => 'languages',
+                ],
+            ],
+        ];
+        $workspaces->loadWorkspaceData($workspace2['id'], $options);
+        $this->_client->writeTableAsyncDirect(
+            $tableId,
+            [
+                'dataWorkspaceId' => $workspace2['id'],
+                'dataObject' => 'languages',
+            ]
+        );
+        // test view is still working
+        $tableRef = $backend->getTableReflection('languages');
+        self::assertEquals(['id', 'name', '_timestamp'], $tableRef->getColumnsNames());
+        self::assertCount(10, $backend->fetchAll('languages'));
+
+//        $fileId = $this->workspaceSapiClient->uploadFile(
+//            (new CsvFile($importFile))->getPathname(),
+//            (new FileUploadOptions())
+//                ->setNotify(false)
+//                ->setIsPublic(false)
+//                ->setCompress(true)
+//                ->setTags(['test-file-1'])
+//        );
+//        // load data from file workspace Not supported yet on S3 on ABS and not used in SNFLK
+//        $fileWorkspace = $workspaces->createWorkspace(
+//            [
+//                'backend' => 'abs',
+//            ],
+//            true
+//        );
+//        $options = [
+//            'input' => [
+//                [
+//                    'dataFileId' => $fileId,
+//                    'destination' => 'languages',
+//                ],
+//            ],
+//        ];
+//        $workspaces->loadWorkspaceData($fileWorkspace['id'], $options);
+//        $this->_client->writeTableAsyncDirect(
+//            $tableId,
+//            [
+//                'dataWorkspaceId' => $fileWorkspace['id'],
+//                'dataObject' => 'languages/',
+//            ]
+//        );
+//        // test view is still working
+//        $tableRef = $backend->getTableReflection('languages');
+//        self::assertEquals(['id', 'name', '_timestamp'], $tableRef->getColumnsNames());
+//        $backend->fetchAll('languages');
+//        self::assertCount(5, $backend->fetchAll('languages'));
+
+        // test drop table
+        $this->_client->dropTable($tableId);
+        $schemaRef = $backend->getSchemaReflection();
+        self::assertCount(0, $schemaRef->getTablesNames());
+        // view is still in workspace but not working
+        self::assertCount(1, $schemaRef->getViewsNames());
+        try {
+            $backend->fetchAll('languages');
+            $this->fail('View should not work after table drop');
+        } catch (Throwable $e) {
+            $this->assertStringContainsString('does not exist or not authorized', $e->getMessage());
+        }
+    }
```

Comment:
Nevím, jestli bysme měli řešit nějaké edge cases - třeba co když si v transformaci to viewčko upravím? Měl by na to mít WS user právo? Řekl bych, že ne. Může ho smazat, může z něj selectovat, ale tím by to mělo hasnout. 

Ale když jsme zjistili, že stejně potřebuje ROIM, tak to moc nepomůže vynucení lineage - dycky to user může workaroundnout tím, že se podívá na tabulku přímo... hmmm, tak nic.